### ISSUE TITLE
VGGSound-Instruments dataset 

### ISSUE THREAD

**Issue by ppx-hub** [hxixixh/mix-and-localize#10]
Thanks for sharing, it's a great job.
However, I didn't find the corresponding training data for the VGGSound-Instruments dataset in the repository, as well as the training script for VGGSound-Instruments. Am I missing something? Looking forward to your reply!

**Comment by IFICL** [comment]
Hi, 
VGGSound-Instruments is a subset of the VGGSound dataset, please refer to Appendix C on page 11 of our paper (https://arxiv.org/abs/2211.15058) for the class categories. 
For the training script, they share the same code with MUSIC dataset, please change the configuration correspondingly.

**Comment by ppx-hub** [comment]
> Hi, VGGSound-Instruments is a subset of the VGGSound dataset, please refer to Appendix C on page 11 of our paper (https://arxiv.org/abs/2211.15058) for the class categories. For the training script, they share the same code with MUSIC dataset, please change the configuration correspondingly.

Received, thanks for the quick reply, what I need to do is to select it from the VGGSound dataset according to the categories you provided (37 categories) and use it as a training set right? Also I would like to know for validation purposes, are the 446 annotated masks you provided included in the 32k video clips of 10s length? Do I need to separate them again?

**Comment by IFICL** [comment]
For the first question, yes. The evaluation dataset we provided is a subset of VGGSound-Instrument. So you need to make sure those are not in the validation or training splits. Unfortunately, we lost the data split file because of disk damage to our server.

**Comment by ppx-hub** [comment]
Thanks for the reply, it's very clear and I have no more questions
